Dockerfile

```diff
@@ -28,14 +27,18 @@ RUN git checkout $(git describe --abbrev=0 --tags $VERSION)
 RUN { [ $(ls /src/modules/ | wc -l) -gt 0 ] && cp -r /src/modules/* /inspircd-src/src/modules/ || echo "No modules overwritten/added by repository"; }
 RUN echo $EXTRASMODULES | xargs --no-run-if-empty ./modulemanager install
 
-RUN ./configure $CONFIGUREARGS --prefix /inspircd --uid 10000 --gid 10000
+RUN ./configure $CONFIGUREARGS --uid 10000 --gid 10000
```

Comment:
Can we make this a separate build argument and add for loop that iterates over them? This would make it a lot more generic to use :)

Reply:
¿Any idea how to split the ARG environment variable and split into multiple commands? I don't find a elegant idea.

Reply:
Have a look at line 28/29 :)

I just realized myself, that xargs is so much easier and works perfect.

`xargs -L1 --no-run-if-empty ./configure`

And you should also swap the commands, since the --prefix is iirc required before you can enable extra modules